runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java

```diff
@@ -124,4 +124,24 @@ default boolean useSmallFont()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		keyName = "infoBoxVertical",
+		name = "Display infoboxes vertically",
+		description = "Toggles the infoboxes displaying vertically"
+	)
+	default boolean infoBoxVertical()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "infoboxmaxrow",
+		name = "Infobox max row count",
+		description = "Configures the amount of infoboxes to show per row"
+	)
+	default int infoboxmaxrow()
```

Comment:
Probably rename this to something different, it is slightly confusing to have "row" for both vertical and horizontal layout.